src/components/atoms/Icon/Icon.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
 import './Icon.scss';
-export default function Image({source, height, width, alt, reqclass}){
-    return <img src={source} alt={alt} className={`${reqclass} `} height={height} width={width} />
+export default function Image({source, height, width, alt, reqclass, onClickHandler}){
```

Comment:
Why does Icon component need on click?

Reply:
In Footer component, I have added an image for changing application locale/language. For that purpose its added.